frontend/src/components/FormControl/FormControl.tsx

```diff
@@ -0,0 +1,18 @@
+import { FormControl } from '@material-ui/core';
+import styled from 'styled-components';
+
+const GreenFormControl = styled(FormControl)({
+  '.MuiOutlinedInput-notchedOutline': {
+    borderColor: 'red',
+  },
+});
+
+const customFormControl = () => {
```

Comment:
komponenta by se určitě měla jmenovat s velkým písmenem na začátku. A držel bych stejný název souboru a název hlavní komponenty v souboru